app/models/manageiq/providers/configuration_manager.rb

```diff
@@ -6,6 +6,7 @@ class ManageIQ::Providers::ConfigurationManager < ::ExtManagementSystem
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
   has_many :inventory_groups,             :dependent => :destroy, :foreign_key => "ems_id", :inverse_of => :manager
+  has_many :inventory_root_groups,        :dependent => :destroy, :foreign_key => "ems_id", :inverse_of => :manager
```

Comment:
well, this is an easy way to define the SQL join, you can do that using arel :-)